Automattic/jetpack
# Jetpack Visual Refresh: debugger

Fixes #29837

## Proposed changes:
* Redesign the Debug page

### Other information:


## Jetpack product discussion
p1HpG7-lJK-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
The new design: #29837

1. Connect Jetpack, scroll down to the bottom menu and click "Debug" (`/wp-admin/admin.php?page=jetpack-debugger`, not to be confused with the "Jetpack Debug" plugin).
2. Confirm the Debug page looks according to the design.
3. Use "Jetpack Debug -> Broken Token" to break the blog token.
4. Go back to the Debug page and confirm that you see the connection errors, and they look according to the design (design pending).
